tech_reports/LLMs/llms.md

```diff
@@ -37,6 +37,17 @@ Authors:
         - [4.10.4.2 Large Matmuls](#41042-large-matmuls)
 
 ## 1. Overview
+What’s the best way to bring up high-performance multi-chip models on Tenstorrent hardware with the Metal stack? We continue to try to answer this question; here are our current best practices, tips and workarounds.
```

Comment:
Chaging the tone of the intro a bit:

This document aims to provide guidance on how to bring up high-performance multi-chip models on Tenstorrent hardware using the TT-Metal stack.
It is targeted at users with previous experience on TT-Metal and shares our current best practices, tips, caveats and workarounds on model bringup.

Reply:
Yes, I agree with comment.